frontend/src/static/js/components/webstatus-sidebar-menu.ts

```diff
@@ -288,6 +293,13 @@ export class WebstatusSidebarMenu extends LitElement {
             >Report an issue</a
           >
         </sl-tree-item>
+
+        <sl-tree-item class="about-item">
+          <sl-icon name="info-circle"></sl-icon>
+          <a class="about-link" href="${ABOUT_PAGE_LINK}" target="_blank"
+            >About Web Platform Status</a
```

Comment:
I was thinking that it would just say "About".  That fits the space better and produces less clutter, which matches the low frequency of use that I would expect of this link.

Reply:
Done.

Reply:
SGTM, and the wiki page should IMO explicitly call out that this dashboard was created by Google, even though that's implicit in the URL (and visible through the affiliation of the main contributors, though that's behind a few clicks). And we should point out the obvious which is that it is now an open source project.

Reply:
If we say the dashboard was created by Google, and at some point there will likely be external contributors, we should probably say it was "initially created by Google".

Reply:
SGTM.